commerce_coordinator/apps/commercetools/management/commands/create_paypal_custom_types.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+from commercetools import CommercetoolsError
+
+from commerce_coordinator.apps.commercetools.catalog_info.foundational_types import TwoUCustomObjects, TwoUCustomTypes
+from commerce_coordinator.apps.commercetools.management.commands._ct_api_client_command import (
+    CommercetoolsAPIClientCommand
+)
+
+
+class Command(CommercetoolsAPIClientCommand):
+    help = "Create the Payment Custom Type/Fields for PayPal"
+
+    def handle(self, *args, **options):
+        current_draft = TwoUCustomTypes.PAYMENT_TYPE_DRAFT
+        type_key = current_draft.key
+        custom_object_draft = TwoUCustomObjects.PAYPAL_CUSTOM_OBJECT_DRAFT
+
+        try:
+            custom_object = self.ct_api_client.base_client.custom_objects.create_or_update(draft=custom_object_draft)
+            print(f"PayPal custom object already exists: {json.dumps(custom_object.serialize())}")
+        except CommercetoolsError as ex:
+            print(f"Error in creating/updating PayPal custom object: {str(ex)}")
+
+        try:
+            ret = self.ct_api_client.base_client.types.get_by_key(type_key)
```

Comment:
Do we have a `create_or_update` method here as well. We can use that to keep code concise and structured in the same way.